docs/history/changelog.md

```diff
@@ -151,6 +151,10 @@ Per the [normative and non-normative content and changes policy](../governance/n
 * [#1209](https://github.com/open-contracting/standard/pull/1209) Replace "electronic goods" with "digital goods" in the description of 'goods' from the `procurementCategory` codelist, to align with the description in the World Trade Organization's Agreement on Government Procurement, and to avoid confusion between electronic goods like computers and digital goods like software.
 * [#1530](https://github.com/open-contracting/standard/pull/1530) Use consistent wording for "goods, services and/or works" in the `classificationScheme`, `extendedProcurementCategory`, `partyRole` and `procurementCategory` codelists.
 * [#1550](https://github.com/open-contracting/standard/pull/1550) Align and improve code descriptions in `extendedProcurementCategory` and `procurementCategory`.
+* [#1645](https://github.com/open-contracting/standard/pull/1645) `relatedProcess.csv`:
+  * Add 'successfulProcess' and deprecate 'prior', 'replacementProcess' and 'renewalProcess'
```

Comment:
```suggestion
  * Deprecate "parent-to-child" codes: 'subContract', 'replacementProcess' and 'renewalProcess'
```

Reply:
After this change, there'll be no mention of 'successfulProcess' in the changelog. Was that intentional?

Reply:
Oh, scratch that. I've just seen https://github.com/open-contracting/standard/pull/1645#discussion_r1414766465